xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/ElementInterface.java

```diff
@@ -68,10 +70,32 @@
      * @param context the XWiki context
      * @param mergeResult the merge report
      * @since 3.2M1
+     * @deprecated Since 14.10.7,15.2RC1 use
```

Comment:
Usually I don't repeat the version since it's now in the `@Deprecated` annotation. I would just write:

`@deprecated use {@link #merge(ElementInterface, ElementInterface, MergeConfiguration, XWikiContext)}.``

WDYT?

Reply:
Fine, I took the habit to specify the version before we had it in the annotation.

Reply:
> WDYT?

That's actually our current rule (but I would not be surprised to see that I did the same mistake by reflex).

Reply:
Yeah, we don't really say it explicitly , but we have https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HDeprecation (I'll update it to mention it)